app/models/group.rb

```diff
@@ -84,11 +93,19 @@ def locked_members
   #
   # @return [Boolean] true if the group can be locked
   def lockable?
-    (members - locked_members).empty? && memberships_count == size
+    (members - locked_members).empty? && members_count == size
   end
 
   private
 
+  # override default attribute getter to include transfers
+  def members_count
+    return 0 unless memberships_count || transfers
+    return memberships_count unless transfers
+    return transfers unless memberships_count
```

Comment:
I don't think we need this; `memberships_count` defaults to 0, and I don't think that we can have a group with transfers without a leader and therefore at least one full member.